src/playlist/PlaylistAny.hxx

```diff
@@ -21,6 +21,7 @@
 #define MPD_PLAYLIST_ANY_HXX
 
 #include "config.h"
+#include "src/LocateUri.hxx"
```

Comment:
Use forward declarations instead of adding an include dependency in a header.

Reply:
Fixed here and in other places.